### DISCUSSION TITLE
旧手机的绳网一直是这个界面

### DISCUSSION THREAD

**Discussion by hjejdjdj** [share121/inter-knot#1476] [其他]
旧手机绳网进不去了，啊啊啊一直是加载界面，开了加速器都没有用，怎么办啊
![Image_1723377038481](https://github.com/user-attachments/assets/c0414a83-782a-4c90-8e9b-3bca20422f06)

**Comment by qwq1949** [comment]
+1qwq
直接在discuss吧qwq
以后说不定会更新了呢qwq
###### 好像看见有的说非会员不能发链接？不知道是不是这个原因...?
